crowbar_framework/app/controllers/repositories_controller.rb

```diff
@@ -0,0 +1,103 @@
+#
+# Copyright 2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class RepositoriesController < ApplicationController
+  before_filter :load_registry
```

Comment:
we load it because it is a class variable, which needs to be loaded into memory to be available

see https://github.com/crowbar/crowbar-core/pull/18#discussion_r38661170 for some discussion.
the other discussion I cannot find anymore, maybe it was in mumble